static/src/javascripts/projects/common/modules/adfree-survey-simple.js

```diff
@@ -0,0 +1,49 @@
+define([
+    'bean',
+    'fastdom',
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/template',
+    'common/views/svgs',
+    'text!common/views/commercial/adfree-survey-simple.html'
+], function (
+    bean,
+    fastdom,
+    $,
+    _,
+    template,
+    svgs,
+    adfreeSurveySimple
```

Comment:
Might be a good idea for the name of the template to be more distinct from the name of the constructor.